README.md

```diff
@@ -21,3 +21,13 @@ Financial services - where we don't allow you to create the strongest password p
 We put punctuation in the list of unusable characters to make it additionally confusing.
 
 ![Williams-Sonoma](/screenshots/williams-sonoma.png "Williams-Sonoma")
+
+### [American Express](https://sso.americanexpress.com/SSO/request?request_type=un_createid&ssolang=en_NL&inav=at_sitefooter_register)
```

Comment:
Thanks for the submission. Can you please alphabetize and resolve any conflicts and I will then merge.